default/web_tt2/edit_list_request.tt2

```diff
@@ -2,297 +2,263 @@
-                 pitem=kS
-                 val=kV
-              %]
-            [%~ END %]
-          [% END %]
-          <!-- end paragraph -->
-          </div>
-
-          [% UNLESS pS.occurrence.match('^1') && pV.size() == 1 ~%] 
-            <div class="small-3 large-2 columns">
-            [% PROCESS EditListArrayDel
-               ppaths=[pS.name,oI]
-               pitem=pS
-               labelled=1
-            %]
-            </div>
-          [%~ END %]
-
-          </div>
-          <span class="divider"></span>
-        [%~ ELSE ~%]
-          <div class="row">
-
-          <div id="item.[% pS.name %].[% oI %]"
-           class="small-9 large-10 columns">
-          [% PROCESS EditListLeaf
-             ppaths=[pS.name,oI]
-             pitem=pS
-             val=oV
-          %]
-          </div>
- 
-          [% UNLESS pS.occurrence.match('^1') && pV.size() == 1 ~%] 
-            <div class="small-3 large-2 columns">
-            [% PROCESS EditListArrayDel
-               ppaths=[pS.name,oI]
-               pitem=pS
-               labelled=1
-            %]
-            </div>
-          [%~ END %]
-
-          </div>
-        [%~ END %]
-        [%~ oI = oI + 1 %]
-      [% END %]
-
-      [%# Additional empty section. ~%]
-      [% IF pS.privilege == 'write' ~%]
-        [% IF pS.type == 'paragraph' ~%]
-          <div class="row">
-
-          <div id="item.[% pS.name %].[% oI %]"
-           class="small-9 large-10 columns">
-          <!-- paragraph -->
-          [% FOREACH kS = pS.format ~%]
-            [% SET kV = kS.default # Default value. ~%]
-
-            [% PROCESS EditListHeading
-               ppaths = [pS.name,oI,kS.name]
-               pitem  = kS
-            ~%]
-
-            [% IF kS.type == 'set' ~%]
-              [% PROCESS EditListSet
-                 ppaths=[pS.name,oI,kS.name]
-                 pitem=kS
-                 val=kV
-              %]
-            [%~ ELSIF kS.type == 'leaf' ~%]
-              [% PROCESS EditListLeaf
-                 ppaths=[pS.name,oI,kS.name]
-                 pitem=kS
-                 val=kV
-              %]
-            [%~ END %]
-          [% END %]
-          <!-- end paragraph -->
-          </div>
-
-          </div>
-        [%~ ELSE ~%]
-          <div class="row">
-
-          <div id="item.[% pS.name %].[% oI %]"
-           class="small-9 large-10 columns">
-          [% PROCESS EditListLeaf
-             ppaths=[pS.name,oI]
-             pitem=pS
-             val=UNDEF # Empty value.
-          %]
-          </div>
-
-          </div>
-        [%~ END %]
-      [%~ END %]
-
-      <!-- end array -->
+    [% IF !group ~%]
+        <h2>[%|loc%]Edit List Configuration[%END%]
+            <a class="openInNewWindow" target="wws_help
+               href="[% 'nomenu/help' | url_rel(['admin-editlist.html']) %]"">
+                <i class="fa fa-info-circle" title="[%|loc%]Help[%END%]" aria-hidden="true"></i>
```

Comment:
Typo.  Should be
``` html
href="[% 'nomenu/help' | url_rel(['admin-editlist.html']) %]">
```

Reply:
Fixed in https://github.com/sympa-community/sympa/pull/582/commits/7b3a39c6ac7c013a0990a7033f81118d1943e2de